lib/dal/src/attribute/value/default_subscription.rs

```diff
@@ -253,7 +216,10 @@ impl AttributeValue {
         ctx: &DalContext,
         id: AttributeValueId,
     ) -> AttributeValueResult<bool> {
-        let category_id = get_or_create_default_subscription_category(ctx).await?;
+        let category_id = ctx
+            .workspace_snapshot()?
+            .get_or_create_static_category_node(CategoryNodeKind::DefaultSubscriptionSources)
```

Comment:
Do these need to do `create`? i.e. if `is_default_subscription_source()` is called on main, should it write to the graph?